Makefile

```diff
@@ -2,13 +2,17 @@
 SHELL:=/bin/bash
 
 # Need Riot installed
-riot:=~/apache-jena-3.8.0/bin/riot
+riot:=/usr/local/Cellar/jena/3.13.1/bin/riot
```

Comment:
@ldragoon I guess we should leave this as 'riot' and force us to add to our path